packages/@okta/courage-for-signin-widget/src/CourageForSigninWidget.js

```diff
@@ -59,6 +59,16 @@ const events = {
 const TextBoxForSigninWidget = TextBox.extend({ events });
 const PasswordBoxForSigninWidget = PasswordBox.extend({ events });
 
+const Form = BaseForm.extend({
+  constructor(options) {
+    // scrollOnError is true by default. Override to false if `scrollOnError` has been set to false in widget options.
+    if (options && options.settings && options.settings.get('features.scrollOnError') === false) {
+      options.scrollOnError = false;
```

Comment:
we can override `scrollOnError` attribute.
```
scrollOnError: function () {
  return options.settings.get('features.scrollOnError') !== false;
}
```

Reply:
Although it's still not idea because we pass SIW option to library, implies the changes are so special to SIW and cannot be port back to Courage.

Reply:
The changes are special to SIW, specifically when the widget is embedded on a page with a fixed header, the scroll logic needs to be adjusted or disabled. The customer is happy with disabled option, which is much easier to implement than a "customize behavior" option.